src/GeositeFramework/js/Export.js

```diff
@@ -105,6 +129,20 @@ require(['use!Geosite'],
                                 .wrapAll("<div class='print-legend-coll'></div>");
                         });
 
+                    function afterPrintHandler() {
+                        _.delay(previewDeferred.resolve, 250);
+                    }
+
+                    $.when(alertDeferred).then(function () {
+                        if (!!window.onafterprint) {//ie, ff, safari?
+                            window.onafterprint = afterPrintHandler;
+                        }
+                        window.print();
```

Comment:
Lines 146 and 147 look suspicious. Were they supposed to be deleted? That might explain the cause behind this issue.